GPU/Vulkan/GPU_Vulkan.cpp

```diff
@@ -327,7 +332,7 @@ void GPU_Vulkan::BuildReportingInfo() {
 	const auto &props = vulkan->GetPhysicalDeviceProperties().properties;
 	const auto &features = vulkan->GetDeviceFeatures().available;
 
-#define CHECK_BOOL_FEATURE(n) do { if (features.n) { featureNames += ", " #n; } } while (false)
+#define CHECK_BOOL_FEATURE(n) do { if (features.features.n) { featureNames += ", " #n; } } while (false)
```

Comment:
Small thing but might be nicer to call it features.standard or something rather than features.features and features.multiviewFeatures.

-[Unknown]